### DISCUSSION TITLE
Dvorak keyboard support (and other layouts like Azerty)

### DISCUSSION THREAD

**Discussion by chris90483** [voxeltycoon/issues#471] [UI&UX]
Hey devs,

I bought the game today and it's great fun! There are definitely things (like the way you can't see vehicles grouped by line for instance) that need improvement but I think the base is solid. It would be great for Dvorak users like myself to see alternative keyboard mappings as a setting in the game, since that would make keybinds and typing a lot easier. I see you can already assign custom keybinds but reassigning them all to an alternative layout is quite a hassle. Thanks for reading and good luck on the development!

**Comment by katepolak** [comment]
A discussion about this has already taken place on the bug tracker, namely here voxeltycoon/bug-tracker#206. Likely the best solution is to add presets for the most common alternative layouts.

**Comment by chris90483** [comment]
@polak-jan correct me if I'm wrong but remapping the keys won't let you type for instance station names in Dvorak right?
